spec/requests/api/v1/admin/account_actions_spec.rb

```diff
@@ -23,6 +23,19 @@
     end
   end
 
+  shared_examples 'a successful logged action' do |action_type|
+    it 'logs action' do
+      subject
+
+      log_item = Admin::ActionLog.last
+
+      expect(log_item).to be_present
+      expect(log_item.action).to eq(action_type)
+      expect(log_item.account_id).to eq(user.account_id)
+      expect(log_item.target_id).to eq(action_type == :disable ? target_account.user.id : target_account.id)
```

Comment:
I think it's not ideal to have this kind of logic here, but that's the best way I found to avoid repeating these lines for every context block.

Reply:
Yeah!

Reply:
Done!